sw/device/tests/BUILD

```diff
@@ -1643,3 +1643,23 @@ opentitan_functest(
         "//sw/device/lib/testing/test_framework:ottf_main",
     ],
 )
+
+opentitan_functest(
+    name = "power_virus_systemtest",
+    srcs = ["power_virus_systemtest.c"],
+    targets = [
+        # TODO(#14814): add more targets
+        "cw310_test_rom",
+        "dv",
+    ],
+    deps = [
+        "//hw/ip/adc_ctrl/data:adc_ctrl_regs",
+        "//hw/top_earlgrey/sw/autogen:top_earlgrey",
+        "//sw/device/lib/dif:adc_ctrl",
+        "//sw/device/lib/dif:gpio",
+        "//sw/device/lib/dif:pinmux",
+        "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing/test_framework:check",
```

Comment:
nit: already pulled in by `ottf_main`.
```suggestion
```

Reply:
hmm that's true, but the `check.h` header is not part of the `ottf_main.h` header, and the test itself uses `CHECK` macros, so this wouldn't follow the "include what you use" principle of the style guide would it? Am I missing something?